### PR TITLE
Download datafile

### PR DESCRIPTION
**Pull request by sabban**
download datafile when cscli hub upgrade if any datafile is missing.

### CODE REVIEW

File: pkg/cwhub/download.go [hunk 1]
```diff
@@ -223,3 +225,44 @@ func DownloadItem(hub *csconfig.Hub, target Item, overwrite bool) (Item, error)
...
+	dec := yaml.NewDecoder(itemFile)
+
+	for {
+		data := &types.DataSet{}
+		err = dec.Decode(data)
+		if err != nil {
+			if err == io.EOF {
+				break
+			} else {
+				return errors.Wrap(err, fmt.Sprintf("while reading file %s", itemFilePath))
```
Comment (buixor) [inline pkg/cwhub/download.go:250]: wrapf instead of wrap + sprintf

File: cmd/crowdsec-cli/utils.go [hunk 2]
```diff
@@ -223,7 +223,11 @@ func UpgradeConfig(itemType string, name string, force bool) {
 		found = true
 		if v.UpToDate {
 			log.Infof("%s : up-to-date", v.Name)
+
 			if !force {
```
Comment (buixor) [inline cmd/crowdsec-cli/utils.go:227]: if force is enabled, it won't download the data files again ? not sure it's what we want

Comment (sabban) [reply cmd/crowdsec-cli/utils.go:227]: it will be download elsewhere anyway. It's to avoid download it twice.

**Review by buixor — APPROVED** [review]
lgtm

**Pull request merged**